requirements_dev.txt

```diff
@@ -1,12 +0,0 @@
-pip==19.2.3
-bump2version==0.5.11
-wheel==0.33.6
-watchdog==0.9.0
-flake8==3.7.8
```

Comment:
I would advise to keep fixed versions of linters and tests (flake8 and other). 

Delete here: Click, tox, twine,  watchdog, bump2version. All thee rest better leave as is.